docs/v3/tutorials/debug.mdx

```diff
@@ -0,0 +1,114 @@
+---
+title: Debug a data pipeline
+description: Find out when flow runs go wrong and how to troubleshoot them.
```

Comment:
```suggestion
description: Learn how to troubleshoot flow runs that error.
```

Reply:
Error is a noun and I think a verb would be more grammatically correct here, so perhaps:

```suggestion
description: Learn how to troubleshoot flow runs that fail.
```

wdyt @discdiver?